src/components/Contentful/LandingPageHero.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<section-with-background :background-content="heroBackground">
+		<template #content>
+			<!-- same wrapper as src/components/BorrowerProfile/ContentContainer.vue -->
```

Comment:
This is almost the same wrapper as `src/components/BorrowerProfile/ContentContainer.vue` Should we move ContentContainer out of BorrowerProfile, and try to make it more reusable since it might be used in other components outside of the borrower profile -- the only difference in this use case is the use of `lg:tw-col-span-6` instead of `lg:tw-col-span-7`

Reply:
My opinion is that the important parts are housed in `kv-page-container` and `kv-grid`. How the columns are used seems more page specific so what you have here seems good to me.